### PR TITLE
Fixes for FHIR-46109: Update AUCDI page to match current AUCDI R1

### PR DESCRIPTION
**Pull request by dt-r**
Fixes [FHIR-46109](https://jira.hl7.org/browse/FHIR-46109) Update AUCDI page to align with AUCDI R1

* Remove stale STU Note 
* Remove rows for Last Administration and Endpoint to match removal from AUCDI
* Add in the rows for new AUCDI elements with TBDs and added comment that this AUCDI element was added to AUCDI and work is underway to map to AU Core.
* Update element name for Body site
* remove references to "AU Core Biomarker"

### CODE REVIEW

File: input/pagecontent/aucdi.md [hunk 1]
```diff
@@ -85,22 +86,28 @@ Column attribute descriptions are as follows:
...
   </tr>
   <tr>
-    <td rowspan="5">Procedure completed event</td>
+    <td colspan="2">Last updated</td>
+    <td>TBD</td>
+    <td>TBD</td>
+    <td>This data element is added to AUCDI R1; work is underway to map to AU Core.</td>
+  </tr>
+  <tr>
+    <td rowspan="5">Procedure completed</td>
```
Comment (dbojicic) [inline input/pagecontent/aucdi.md:95]: It is actually "Procedure completed event"; sorry - I think I changed this one. Please revert.

File: input/pagecontent/aucdi.md [hunk 2]
```diff
@@ -60,14 +55,20 @@ Column attribute descriptions are as follows:
...
+  <tr>
+    <td rowspan="5">Problem/Diagnosis summary</td>
     <td colspan="2">Problem / Diagnosis name</td>
     <td><a href="StructureDefinition-au-core-condition.html">AU Core Condition</a></td>
     <td>Condition.code</td>
     <td></td>
   </tr>
   <tr>
-    <td colspan="2">Body site</td>
+    <td colspan="2">Body site/laterality</td>
```
Comment (dbojicic) [inline input/pagecontent/aucdi.md:71]: Sorry - I think this was me, could you please change back to 'Body site'. Thanks.

**Review by dbojicic — COMMENTED** [review]
Just a few tweaks.

**Review by dbojicic — CHANGES_REQUESTED** [review]
Just a few tweaks.

File: input/pagecontent/aucdi.md (hunk 2 above)
Comment (dt-r) [reply input/pagecontent/aucdi.md:71]: Latest version I have access to says Body site/laterality. This stays for now. No change until final AUCDI is published.

File: input/pagecontent/aucdi.md [hunk 3]
```diff
@@ -220,13 +234,6 @@ Column attribute descriptions are as follows:
...
-  <tr>
-    <td>Tobacco smoking summary</td>
-    <td colspan="2">Overall status</td>
-    <td><a href="StructureDefinition-au-core-smokingstatus.html">AU Core Smoking Status</a></td>
-    <td>Observation</td>
-    <td></td>
-  </tr>
   <tr>
     <td rowspan="3">Encounter – clinical context</td>
     <td colspan="2">Reason for encounter</td>
```
[the remark this answers is not in the document]
Comment (dt-r) [reply input/pagecontent/aucdi.md:239]: Refactored, that row should not have been present. The mappings in that row were for modality.

File: input/pagecontent/aucdi.md (hunk 1 above)
Comment (dt-r) [reply input/pagecontent/aucdi.md:95]: At this point it's 50/50. I'll make the change and we'll re-evaluate when AUCDI R1 is published.

File: input/pagecontent/aucdi.md [hunk 4]
```diff
@@ -183,26 +203,20 @@ Column attribute descriptions are as follows:
...
   </tr>
   <tr>
-    <td rowspan="3">Sex and gender</td>
+    <td colspan="2">Date of assertion</td>
+    <td>TBD</td>
+    <td>TBD</td>
+    <td>This data element is added to AUCDI R1; work is underway to map to AU Core.</td>
+  </tr>
+  <tr>
+    <td rowspan="3">Sex and gender summary</td>
```
[the remark this answers is not in the document]
Comment (dt-r) [reply input/pagecontent/aucdi.md:219]: Added.

**Pull request merged**